examples/jsonExample/src/main/java/io/realm/examples/json/JsonExampleActivity.java

```diff
@@ -44,9 +44,10 @@
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
+        Realm.init(getApplicationContext());
```

Comment:
It was mostly to avoid having to create the MyApplication subclass, but I was probably just beeing lazy, and I agree that this pattern is bad. I'll add the Application subclass to the other examples.